CHANGES.md

```diff
@@ -1,12 +1,14 @@
-## pending
+# pending
 
 Light Client:
 
 - Expose latest_trusted from Supervisor Handle ([#394])
 - Turn `Handle` into a trait for ease of integration and testability ([#401])
+- Improve `Supervisor` ergonomics ([#403])
```

Comment:
Not sure what others think about this but if there is a realted ADR, it be cool to link it in the changelog, too.